tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -199,20 +199,14 @@ bool TypeTranslator::isScalarType(QualType type, QualType *scalarType) {
 
 bool TypeTranslator::isRWByteAddressBuffer(QualType type) {
   if (const auto *rt = type->getAs<RecordType>()) {
-    const auto &rtName = rt->getDecl()->getName();
-    if (rtName == "RWByteAddressBuffer") {
-      return true;
-    }
+    return rt->getDecl()->getName().compare("RWByteAddressBuffer") == 0;
```

Comment:
Can directly use `==` to compare actually. `operator==` is a wrap around `StringRef::compare`. :)